imgproc/opencv_optical_flow.h

```diff
@@ -68,16 +68,16 @@ class OpenCVOpticalFlow
 
     void render(cv::Mat &outputImage, unsigned int scale)
     {
-        assert(outputImage.cols == m_FlowX.cols * scale);
-        assert(outputImage.rows == m_FlowX.rows * scale);
+        assert(outputImage.cols == m_FlowX.cols * (int) scale);
```

Comment:
maybe just change ``scale`` to an ``int`` and add ``assert(scale >= 1)`` - using ``unsigned int`` is a bad habit of mine

Reply:
Ok, will do